Source/SpacePort/SpacePort/Menu.cs

```diff
@@ -1,39 +1,255 @@
+                    {
+                        Console.ForegroundColor = ConsoleColor.Red;
+                        Console.WriteLine("You’re fulfilling your destiny, Traveler. Become my apprentice. Learn to use the Dark Side of the Force.");
+                        Console.ResetColor();
+                        Console.WriteLine("Please press any key to return to main menu");
+                        Console.ReadKey();
+                        MenuChoice = 0;
+                    }                
+                }
+                else if (MenuChoice == 2)
+                {
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine("\n\rI find your lack of faith disturbing.\n\r");
+                    Console.ResetColor();
+
+                    Console.Write("To check out, please enter your name: ");
+                    string name = Console.ReadLine();
+
+                    try
+                    {
+                        Dictionary<string, double> ships = GetShips();
+                        spacePort.Undock(ships[shipName]);
+                        spacePort.DisplayReceipt(name);
+                    }
+                    catch (NullReferenceException)
+                    {
+                        Console.WriteLine("Please Check in before Check out");
+                    }
+
+
+                    Console.WriteLine("\n\rPlease press any key to return to main menu");
+                    Console.ReadKey();
+                    Console.Clear();
+                    
+                    
+                    MenuChoice = 0;
+                    Console.WriteLine("Please press Enter to start the program");
+                }
+                else
+                {
+                    Console.Clear();
+                    Console.WriteLine("Please enter a valid menu choice number");
+                    MenuChoice = 0;                    
+                }
+            }
         }
 
-        public static void DisplayShipTypes()
+        private static void DisplayMain()
         {
-            //List<string> ships = api.ShipData();
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Hello, my friend. Stay awhile and listen");
+            Console.WriteLine("I am A.S.P.P.A:  *BEEP* *BEEP*  Your Automated SpacePort Parking Assistent \n\r");
+            Console.ResetColor();
 
-            /*foreach (var s in ships)
+            Console.WriteLine("[1\tCheck In\t]");
+            Console.WriteLine("[2\tCheck Out\t]");
+        }
+        private static DateTime InputDate(string input)
+        {
+            DateTime dateTime = DateTime.Parse(input);
+
+            return dateTime;
+        }
+
+        private static bool ValidateTraveler()
+        {
+            jObject = swAPI.GetTravelerDataAsync(travelerName).Result;
+            bool isValid = swAPI.ValidateTraveler(jObject, travelerName);
+            return isValid;
+        }
```

Comment:
Method does exactly what it says and delegates work down to API very nice
Could possible look like:

            jObject = swAPI.GetTravelerDataAsync(travelerName).Result;
            return swAPI.ValidateTraveler(jObject, travelerName);